csharp/device/samples/UWPSample/project.lock.json

```diff
@@ -11523,7 +11523,7 @@
       ]
     },
     "Microsoft.ApplicationInsights.WindowsApps/1.0.0": {
-      "sha512": "NvBQnFeiFd0O1QdBz06UGApD7zn7ztVi7qO18IsM3EjiXRNgfrEBXB+azNm8XqLY8xGFAqh3HAuSd/wHZMe0XA==",
```

Comment:
Those are added by VS as soon as you build the UWP project.
If they are not in the original branch they probably should.
If you don't agree feel free to remove them.